plugins/arDominionB5Plugin/modules/sfIsaarPlugin/templates/indexSuccess.php

```diff
@@ -60,201 +60,209 @@
 
-  <?php echo link_to_if(QubitAcl::check($resource, 'update'), '<h2>'.__('Identity area').'</h2>', [$resource, 'module' => 'actor', 'action' => 'edit'], ['anchor' => 'identity-collapse', 'title' => __('Edit identity area')]); ?>
+    <?php if (0 < count($resource->digitalObjectsRelatedByobjectId)) { ?>
+      <?php echo get_component('digitalobject', 'show', ['link' => $digitalObjectLink, 'resource' => $resource->digitalObjectsRelatedByobjectId[0], 'usageType' => QubitTerm::REFERENCE_ID]); ?>
+    <?php } ?>
 
-  <?php echo render_show(__('Type of entity'), render_value($resource->entityType)); ?>
+    <section id="identityArea" class="border-bottom">
 
-  <?php echo render_show(__('Authorized form of name'), render_value($resource->getAuthorizedFormOfName(['cultureFallback' => true]))); ?>
+      <?php echo link_to_if(QubitAcl::check($resource, 'update'), render_b5_section_label(__('Identity area')), [$resource, 'module' => 'actor', 'action' => 'edit'], ['anchor' => 'identity-collapse', 'title' => __('Edit identity area'), 'class' => 'text-primary']); ?>
 
-  <div class="field">
-    <h3><?php echo __('Parallel form(s) of name'); ?></h3>
-    <div>
-      <ul>
-        <?php foreach ($resource->getOtherNames(['typeId' => QubitTerm::PARALLEL_FORM_OF_NAME_ID]) as $item) { ?>
-          <li><?php echo render_value_inline($item->__toString()); ?></li>
-        <?php } ?>
-      </ul>
-    </div>
-  </div>
+      <?php echo render_show(__('Type of entity'), render_value_inline($resource->entityType)); ?>
 
-  <div class="field">
-    <h3><?php echo __('Standardized form(s) of name according to other rules'); ?></h3>
-    <div>
-      <ul>
-        <?php foreach ($resource->getOtherNames(['typeId' => QubitTerm::STANDARDIZED_FORM_OF_NAME_ID]) as $item) { ?>
-          <li><?php echo render_value_inline($item->__toString()); ?></li>
-        <?php } ?>
-      </ul>
-    </div>
-  </div>
+      <?php echo render_show(__('Authorized form of name'), render_value_inline($resource->getAuthorizedFormOfName(['cultureFallback' => true]))); ?>
 
-  <div class="field">
-    <h3><?php echo __('Other form(s) of name'); ?></h3>
-    <div>
-      <ul>
-        <?php foreach ($resource->getOtherNames(['typeId' => QubitTerm::OTHER_FORM_OF_NAME_ID]) as $item) { ?>
-          <li><?php echo render_value_inline($item->__toString()); ?></li>
-        <?php } ?>
-      </ul>
-    </div>
-  </div>
+      <div class="field <?php echo render_b5_show_field_css_classes(); ?>">
+        <?php echo render_b5_show_label(__('Parallel form(s) of name')); ?>
+        <div class="<?php echo render_b5_show_value_css_classes(); ?>">
+          <ul class="<?php echo render_b5_show_list_css_classes(); ?>">
+            <?php foreach ($resource->getOtherNames(['typeId' => QubitTerm::PARALLEL_FORM_OF_NAME_ID]) as $item) { ?>
+              <li><?php echo render_value_inline($item->__toString()); ?></li>
+            <?php } ?>
+          </ul>
+        </div>
+      </div>
```

Comment:
One suggestion about what we chatted yesterday (`sfOutputEscaper::unescape()` may not be needed in some cases) ...

```php
<?php echo render_show(__('Parallel form(s) of name'), sfOutputEscaper::unescape(
    $resource->getOtherNames(['typeId' => QubitTerm::PARALLEL_FORM_OF_NAME_ID])
)); ?>
```

And in the helper ...

```php
function render_b5_show_value($value, $options = [])
{
    $tag = 'div';
    $cssClasses = render_b5_show_value_css_classes($options);

    $finalValue = $value;
    if (is_array($value)) {
        $finalValue = '<ul class="'.render_b5_show_list_css_classes().'">';
        foreach ($value as $item) {
            $finalValue .= '<li>'.render_value_inline(string($item)).'</li>';
        }
        $finalValue .= '</ul>';
    }

    return render_b5_show_container($tag, $finalValue, $cssClasses, $options);
}
```